code/game/objects/items/shards.dm

```diff
@@ -172,7 +172,7 @@
 		playsound(M, 'sound/effects/meteorimpact.ogg', 35)
 		M.at_munition_interior_explosion_effect(cause_data = create_cause_data("Anti-Tank Rocket", U))
 		M.interior_crash_effect()
-		M.ex_act(1000, get_dir(U, T), create_cause_data("Anti-Tank Rocket", U))
+		M.ex_act(1000)
```

Comment:
I don't get this change, are you removing the direction to stop throwing?

Reply:
ex_act doesn't take these arguments. 
![image](https://github.com/user-attachments/assets/d56ffac9-f2d6-44ef-8c26-f793b059f02b)